google/generativeai/files.py

```diff
@@ -35,7 +35,28 @@ def upload_file(
     mime_type: str | None = None,
     name: str | None = None,
     display_name: str | None = None,
+    resumable: bool | None = None,
+    chunksize: float | None = None,
```

Comment:
Should `chunksize` be a float? Docs indicate it's "bytes", so unless we support bit-level chunks I would have thought `int`

Reply:
ah yeah int is the right param type. fixed thanks for catching that